documentation/faq.mdx

```diff
@@ -77,6 +77,14 @@ The work done on OpenGOAL will not be a big help for decompiling anything outsid
 
 Most importantly, we are motivated on working on Jak, not other games. Those other games likely already have small niche communities plugging away at things in the background just like we were.
 
+### What are these few files left to decompile?
+
+The files left for decompilation as shown with inconsistent colors at the non-developer progress overviews are not needed for the game to run. It's mostly debug-only code, PS2 kernel debug stuff, renderer code that is already reimplemented in C++ or code used for recording bot paths etc. We do not intend to decompile them.
+
+<!-- https://discord.com/channels/756287461377703987/1094315000308437162/1233981481509195837 -->
+<!-- https://discord.com/channels/756287461377703987/756287461830688810/1251236663200448543 -->
+<!-- https://discord.com/channels/756287461377703987/756287461830688810/1273233345106673698 -->
```

Comment:
I wouldn't link to discord threads, if there is something important from them I would summarize it.

Also I don't think this is the best location, better to put an explanation on the actual progress page, rather than the FAQ, as people never read the FAQ.

Reply:
I agree. I had made this commit when this topic was brought up _again_, but didn't see it reappear ever since. I always add the Discord links to “prove” they're questions people ask, or to easily find the “official” answers -- that I have a hard time finding usually.